### ISSUE TITLE
Simulation ran into infinite loop with same vehicle. 

### ISSUE THREAD

**Issue by fayexin** [matsim-org/matsim-code-examples#590]
I am running the simulation with both cars and public transportation.  The simulation now runs into infinite loop with the same vehicle, the log file is like, 

2021-07-19T16:11:30,686  INFO AbstractQNetsimEngine:365 SIMULATION (QNetsimEngine) AT 87:00:00 : #links=0 #nodes=0
2021-07-19T16:11:30,687  INFO QSim:542 SIMULATION (NEW QSim) AT 87:00:00 : #Veh=2 lost=0 simT=301685.0s realT=45s; (s/r): 6704.111111111111
2021-07-19T16:11:30,807  INFO AbstractQNetsimEngine:365 SIMULATION (QNetsimEngine) AT 88:00:00 : #links=0 #nodes=0
2021-07-19T16:11:30,807  INFO QSim:542 SIMULATION (NEW QSim) AT 88:00:00 : #Veh=2 lost=0 simT=305285.0s realT=45s; (s/r): 6784.111111111111
2021-07-19T16:11:30,910  INFO AbstractQNetsimEngine:365 SIMULATION (QNetsimEngine) AT 89:00:00 : #links=0 #nodes=0
2021-07-19T16:11:30,910  INFO QSim:542 SIMULATION (NEW QSim) AT 89:00:00 : #Veh=2 lost=0 simT=308885.0s realT=45s; (s/r): 6864.111111111111
2021-07-19T16:11:31,008  INFO AbstractQNetsimEngine:365 SIMULATION (QNetsimEngine) AT 90:00:00 : #links=0 #nodes=0
2021-07-19T16:11:31,008  INFO QSim:542 SIMULATION (NEW QSim) AT 90:00:00 : #Veh=2 lost=0 simT=312485.0s realT=45s; (s/r): 6944.111111111111

What caused this problem? Thanks a lot.

**Comment by Janekdererste** [comment]
From the output you've posted, it seems like there are 2 vehicles conducting trips for a very long time. 

Since you've mentioned pt being part of your set up, you could check whether you have pt trips in your transit schedule at this time. Also, you could check your plans file, whether it contains persons with activities ending or starting at these late times. 

If you don't care about these two vehicles, you can also specify a simulation end time. 
```
<module name="qsim">
        <param name="endTime" value="36:00:00"/>
        ....
</module>
```

**Comment by fayexin** [comment]
Thanks a lot! I have changed the ending time in config file and It didn't give me a very long trip conduct forever. 

 I wan to find which vehicle caused those problems. I manually located several passengers who would have those long trips even when they are the only person in the plan file. Since I could only get the 0 iteration, their plan file looks very normal to me. I am not sure how to figure out which may caused problem. 

Thanks a lot for your help!
[0.plans.xml.gz](https://github.com/matsim-org/matsim-code-examples/files/6848537/0.plans.xml.gz)
